packages/backend/src/modules/activity/services/providers/RpcTxsCountProvider.ts

```diff
@@ -17,15 +19,24 @@ export class RpcTxsCountProvider {
     to: number,
   ): Promise<ActivityRecordWithoutRatio[]> {
     const queries = range(from, to + 1).map(async (blockNumber) => {
-      const block = await this.rpcClient.getBlock(blockNumber)
+      const block = await this.rpcClient.getBlockWithTransactions(blockNumber)
+      const { transactionsLength, uopsLength } =
+        this.uopsAnalyzer.analyzeBlock(block)
       const timestamp = new UnixTime(block.timestamp)
 
+      const txsCount =
+        this.assessCount?.(transactionsLength, blockNumber) ??
+        transactionsLength
+      const uopsCount =
+        this.assessCount?.(uopsLength, blockNumber) ?? uopsLength
+
+      const ratio = uopsCount / txsCount
```

Comment:
Let's remove ration calculation from here as it's no being used and return types is `ActivityRecordWithoutRatio`